src/Classes/ServiceAPI/MyRadio_Demo.php

```diff
@@ -343,14 +361,13 @@ public function leave()
             'Training Cancellation',
             'Hi ' . $attendee->getFName() . ",\r\n\r\n"
                 . "Just to confirm that you have left the training session at " . $this->getDemoTime() . ". If this was accidental, simply rejoin. "
-                . "Meanwhile, you've been added to the waiting list, and we'll let you know if a session becomes available."
+                . "Meanwhile, you've can join the waiting list, and we'll let you know if a session becomes available. "
```

Comment:
Err, hang on - why are you telling them to join the waiting list when they've just left a sesh?

Reply:
If they've left cause they can't make the time, doesn't necessarily mean they don't want the training.

Reply:
> you've can join the waiting list

Reply:
https://github.com/UniversityRadioYork/MyRadio/pull/934/commits/3aee163c4db5c963851d5356654e4bc1ac2ef90c#diff-2532bcdab17654f155a88878e22e5c7665fa6ffcf2e813d4a306a5473801cc9bR368